src/custom-types.json

```diff
@@ -583,6 +583,13 @@
       "http://facebook.github.io/react/docs/getting-started.html"
     ]
   },
+  "text/vue": {
+    "compressible": true,
+    "extensions": ["vue"],
+    "sources": [
+      "http://vuejs.org/v2/guide/single-file-components.html"
```

Comment:
Please provide a primary source for the mapping of the mime type `text/vue` to the extension `.vue`. I don't see the mime type anywhere on that page.